WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/history/ScanHistoryListFragment.kt

```diff
@@ -68,10 +69,24 @@ class ScanHistoryListFragment : ViewPagerFragment(R.layout.scan_history_list_fra
         }
     }
 
+    private fun getTabType(): ScanHistoryTabType =
+            requireNotNull(arguments?.getParcelable(ARG_TAB_TYPE))
```

Comment:
Minor (🔍): This can become a one-liner.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13827/commits/c203e55da6ebf1c4921c38bd862250836a134a9b